src/handlers/channel.ts

```diff
@@ -325,9 +325,13 @@ export function editChannel(
 
   const payload = {
     ...options,
+    // deno-lint-ignore camelcase
```

Comment:
I think if we move this payload entirely into the function call below and not in a variable deno lint will not have errors on this. It will also slightly improve our performance as we don't need to declare this const